tests/core/editable/inserthtmlintospan.js

```diff
@@ -0,0 +1,33 @@
+/* bender-tags: editor,insertion */
```

Comment:
Do we really need a separate file for these tests? It looks like they could be placed inside `tests/core/editable/insertion.js`. WDYT?